.github/workflows/node-zxc-compile-application-code.yaml

```diff
@@ -516,7 +516,9 @@ jobs:
             ) &&
             !cancelled()
           }}
-        run: sed -i 's/^org.gradle.configuration-cache=.*$/org.gradle.configuration-cache=false/' gradle.properties
+        run: |
+          sed -i 's/^org.gradle.configuration-cache=.*$/org.gradle.configuration-cache=false/' gradle.properties
+          touch build.gradle.kts
```

Comment:
Create empty `build.gradle.kts` in root, because Snyk right now expects a Gradle project to have a this. Which is not a requirement for a Gradle project, only `settings.gradle.kts` is required.